python/setup.cfg

```diff
@@ -1,11 +1,35 @@
 # Copyright (c) 2018, NVIDIA CORPORATION.
 
 [flake8]
-exclude = cuML,ml-prims,__init__.py,versioneer.py
+filename = *.py, *.pyx, *.pxd
+exclude = 
+   cpp,
+   thirdparty,
+   versioneer.py,
+   *.egg,
+   build/,
+   # docs,
+   .git,
+   __pycache__
+
+# Cython Rules ignored:
+# E999: invalid syntax (works for Python, not Cython)
+# E225: Missing whitespace around operators (breaks cython casting syntax like <int>)
+# E226: Missing whitespace around arithmetic operators (breaks cython pointer syntax like int*)
+# E227: Missing whitespace around bitwise or shift operator (Can also break casting syntax)
+# W503: line break before binary operator (breaks lines that start with a pointer)
+# W504: line break after binary operator (breaks lines that end with a pointer)
+
+per-file-ignores = 
+    # imported but unused
+    __init__.py: F401
+    # Cython Exclusions
+    *.pyx: E999, E225, E226, E227, W503, W504
```

Comment:
Ah, nice, seems like this per-file-ignores is the magic that fixes things up.

Reply:
Exactly. Dont need to invoke flake8 twice now with 2 different configs.